packages/quantic/jest.config.js

```diff
@@ -13,11 +13,16 @@ module.exports = {
   ],
   moduleNameMapper: {
     '^@salesforce/i18n/': '<rootDir>/typings/lwc/customlabels.d.ts',
-    '^lightning/modal$': '<rootDir>/force-app/test/jest-mocks/lightning/modal/modal',
-    '^lightning/modalBody$': '<rootDir>/force-app/test/jest-mocks/lightning/modalBody/modalBody',
-    '^lightning/modalFooter$': '<rootDir>/force-app/test/jest-mocks/lightning/modalFooter/modalFooter',
-    '^lightning/modalHeader$': '<rootDir>/force-app/test/jest-mocks/lightning/modalHeader/modalHeader',
-    '^c/quanticResult$': '<rootDir>/force-app/test/jest-mocks/quantic/quanticResult/quanticResult',
```

Comment:
We shouldn't delete this, it's needed for the the unit tests of the Quantic Result Children component.